src/main/webapp/app/overview/course-dashboard/course-dashboard.component.html

```diff
@@ -0,0 +1,58 @@
+@if (irisEnabled) {
+    <div class="chat-container">
+        <jhi-course-chatbot [courseId]="courseId" />
+    </div>
+}
+<div class="stats-container">
+    @if (isLoading) {
+        <div class="d-flex justify-content-center">
+            <div class="spinner-border" role="status">
+                <span class="sr-only">{{ 'loading' | artemisTranslate }}</span>
+            </div>
+        </div>
+    } @else {
+        @if (hasExercises) {
+            @if (points > 0 || maxPoints > 0) {
+                <div class="d-sm-flex gap-3 align-items-baseline">
+                    <h4 jhiTranslate="artemisApp.courseStudentDashboard.overallPerformance.title"></h4>
+                    <div class="flex-grow-1">
+                        <ngb-progressbar type="success overflow-visible" [value]="points" [max]="maxPoints" height="1.75rem">
+                            <span class="fs-5 mx-2">{{ points }} / {{ maxPoints }} ({{ round((points / maxPoints) * 1000) / 10 }}%)</span>
```

Comment:
Would it be possible to compute the rounded value in the component ts? That way, we'd avoid the call to `round`.